sqlbrite/src/main/java/com/squareup/sqlbrite/BriteDatabase.java

```diff
@@ -547,6 +549,42 @@ public void executeAndTrigger(String table, String sql, Object... args) {
     sendTableTrigger(Collections.singleton(table));
   }
 
+  /**
+   * Execute {@code statement}, if the the number of rows affected by execution of this SQL
+   * statement is of any importance to the caller - for example, UPDATE / DELETE SQL statements.
+   *
+   * @return the number of rows affected by this SQL statement execution.
+   * @throws android.database.SQLException If the SQL string is invalid
+   *
+   * @see SQLiteStatement#executeUpdateDelete()
+   */
+  @RequiresApi(api = Build.VERSION_CODES.HONEYCOMB)
```

Comment:
You can omit the `api = ` part. I'm not sure why they included both that and `value`.